doc/build_apps/get_started.rst

```diff
@@ -0,0 +1,172 @@
+-  :doc:`Build new CCF applications </build_apps/index>` in TypeScript/JavaScript or C++
+-  CCF `Modules API reference <https://microsoft.github.io/CCF/main/js/ccf-app/modules.html>`__
+-  CCF application get started repos `CCF application template <https://github.com/microsoft/ccf-app-template>`__ and  `CCF application samples <https://github.com/microsoft/ccf-app-samples>`__
+
+Development environment
+-----------------------
+
+-  VS Code Dev Container |Open in VSCode|
+-  Github codespace: |Github codespace|
+-  Virtual Machine (`Creating a Virtual Machine in Azure to run CCF <https://github.com/microsoft/CCF/blob/main/getting_started/azure_vm/README.md>`__)
+
+(JavaScript/Typescript) Applications
+------------------------------------
+
+CCF apps can be written in JavaScript or Typescript, To test a JS/TS CCF application you need go through the following steps:
+
+-  :doc:`Start a CCF Network with at least one node </operations/start_network>`
+-  Initialize the CCF network with at least one (active member - user) :ref:`governance/open_network:Opening a Network`,
+   this can be done through :ref:`Proposals <governance/proposals:Summary>`.
+-  Create an application :ref:`deployment proposal <build_apps/js_app_bundle:Deployment>`
+-  Submit the app deployment proposal to the network and all members accept it through voting. This is a part of :ref:`Network Governance <governance/proposals:Submitting a New Proposal>`.
+-  :doc:`Open the CCF network for users </governance/open_network>`
+-  Start testing of your application endpoints
+
+Build Application
+~~~~~~~~~~~~~~~~~
+
+The application building prerequisites [:doc:`CCF </build_apps/install_bin>`, `NodeJS <https://nodejs.org>`__ and `NPM <https://www.npmjs.com>`__] must be installed, all will be preinstalled if you are using devcontainer environment, otherwise you need to install them manually.
+
+Please follow `ccf-app-template build process <https://github.com/microsoft/ccf-app-template#run-js-app>`__
+
+Testing your Application
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several approaches to run and test your application
+
+Run app: Using Sandbox.sh
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  Running the sandbox.sh script automatically starts a CCF network and deploys your application on it. The app is up and ready to receive calls and the initial governance steps are done for you
+-  Support both ccf network types [virtual - enclave (TEE hardware)]
+-  No initial governance steps required
+-  How is it working?
+
+   -  :doc:`/build_apps/run_app`
+   -  `CCF Application template repository <https://github.com/microsoft/ccf-app-template#run-js-app>`__
+
+Run app: Using Docker
+^^^^^^^^^^^^^^^^^^^^^
+
+-  Build and run one of these docker files [“ccf_app_js.virtual” or “ccf_app_js.enclave”] to start a CCF network with one node and one member.
+-  Support both ccf network types [virtual - enclave (TEE hardware)]
```

Comment:
Because these docker images are only available on the `ccf-app-template` repository, I think it's preferable not to include them there.

Reply:
Rephrased: "A CCF network can be started using Docker containers; please check the docker file samples"